i18n/de/Amethyst.ftl

```diff
@@ -1,158 +1,161 @@
 # used across files
-exiting = Exiting
+exiting = Schließt...
 packages = { $pkgNum ->
-    [one] package
-    *[other] packages
+    [one] Paket
+    *[other] Pakete
 }
 missing-deps = Fehlende Packetabhängigkeiten
-continue = Fortführen?
+continue = Fortfahren?
 # main
-run-as-root = Amethyst als root auszuführen ist nicht erlaubt und kann zu kaputten systemen führen. Amethyst wird sie statdessen nach root rechten fragen wenn diese nötig sind.
+run-as-root = Amethyst  mit root Berechtigungen auszuführen ist nicht erlaubt und kann zu korrupten Systemen führen. Amethyst wird nach root Berechtigungen fragen, wenn diese nötig sein sollten.
 following-packages = Folgende Pakete wurden in der AUR gefunden:
 aur-warning = 
-    Die AUR ist eine Quelle an Paketen die von den benutzern hochgeladen werden, welche nicht immer sicher sind.
-    Es wird empfohlen die PKGBUILD der Pakete zu überprüfen bevor sie diese Pakete installieren, da manche PKGBUILDs gefährlich sein können. 
-    Diese Warnung kann in der Konfigurationsdatei abgeschaltet werden.
-are-you-sure = Sind sie sich sicher, dass sie fortfahren möchten?
+    Die AUR ist eine Plattform, welche von der Community verwaltete Pakete zur Verfügung stellt und die Sicherheit eines Paketes nicht gewährleisten kann.
+    Es wird empfohlen die PKGBUILD-Dateien der Pakete zu überprüfen bevor Du diese Pakete installierst, da manche PKGBUILD-Dateien gefährlich sein können. 
```

Comment:
I would use `bevor sie diese Pakete installieren` to make it more formal

Reply:
I don't want to be formal tho

Reply:
why not?

Reply:
i dont like formality

Reply:
> why not?
> 

do we *need* to be?

Reply:
yes

Reply:
> why not?

Because ame is a fresh new package manager, not enterprise software.

Reply:
> yes
> 

no we dont formality is weird and this aint a program made by a billion dollar corporation

Reply:
But we don't have to be enterprise software to make it formal, a lot of software with german translation uses the `sie` form for stuff, it's generally the way we should refer to the user in software

Reply:
> But we don't have to be enterprise software to make it formal, a lot of software with german translation uses the `sie` form for stuff, it's generally the way we should refer to the user in software

No I don't want to refer to the user as `Sie`. I feel like I'm exclusively referring to 50+ year olds this way.

Reply:
Maybe you are? You don't know who's using the software, which is why we should use `Sie`

Reply:
> Maybe you are? You don't know who's using the software, which is why we should use `Sie`

IKEA uses Du too
So let's compromise and introduce a DE_FORMAL Language code

Reply:
Having formality in a program can make it seem like better software to some people. If the translation seems professional, then that gives the impression that the software is also written professionally. Plus, in French conventions at least, that's just a more polite way to refer to someone.

Reply:
I would prefer to not add a new language code just because of one disagreement, it can cause many issues and most likely won't be properly used

Reply:
> Having formality in a program can make it seem like better software to some people. If the translation seems professional, then that gives the impression that the software is also written professionally. Plus, in French conventions at least, that's just a more polite way to refer to someone.

All software I know that uses formal language contains the most horrible crimes in the source code

Reply:
Ok, but how does that argue against having formal (and more polite) ways of reffering to the user?

Reply:
> Ok, but how does that argue against having formal (and more polite) ways of reffering to the user? It's not like having some way of reffering to the user will automatically make the code bad

I'm just saying that I don't think the "quality" of the application feels worse just because we're not using formal language. And to me it's the opposite because of past experience where applications that don't use formal language feel more polished

Reply:
I'd certainly like a program more if it refers to me with formal/polite language and I don't think I'm the only one with that

Reply:
> I'd certainly like a program more if it refers to me with formal/polite language and I don't think I'm the only one with that

Können Sie mir das schriftlich bestätigen?